src/data/metadata.csv

```diff
@@ -9053,6 +9052,741 @@ IN_TR,IN,India,TR,Tripura,,,,,,
 IN_UP,IN,India,UP,Uttar Pradesh,,,,,,
 IN_UT,IN,India,UT,Uttarakhand,,,,,,
 IN_WB,IN,India,WB,West Bengal,,,,,,
+IN_AN_NMA,IN,India,AN,Andaman and Nicobar Islands,NMA,North and Middle Andaman,,,North and Middle Andaman,
```

Comment:
Please sort these lexicographically (i.e. the order should be `IN_AN`, `IN_AN_NMA`, ..., `IN_AP`, `IN_AP_ANT`, ...)

Reply:
Done.